EPRN.Common.Data/Migrations/20240214120931_ReasonNowNotRequired.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
When you see to my comment above, delete this migration and re-generate

Reply:
Done.